core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -557,6 +560,42 @@ case object ModelInsights {
             case _ => None
           }
           val keptIndex = indexInToIndexKept.get(h.index)
+          val featureStd = getIfExists(h.index, s.featuresStatistics.variance).getOrElse(1.0)
+          val sparkFtrContrib = keptIndex
+            .map(i => contributions.map(_.applyOrElse(i, (_: Int) => 0.0))).getOrElse(Seq.empty)
+          val LRStandardization = checkLRStandardization(model).getOrElse(false)
+          val defaultLabelStd = 1.0
+          val labelStd = label.distribution match {
+            case Some(Continuous(_, _, _, variance)) =>
+              if (variance == 0) {
+                log.warn("The standard deviation of the label is zero, " +
+                  "so the coefficients and intercepts of the model will be zeros, training is not needed.\"")
+                defaultLabelStd
+              }
+              else math.sqrt(variance)
```

Comment:
are these formulas (here and on line 582) from the spark regularization? and if so does it work for all model types?

Reply:
this and line 582 are calculating standard deviation of the label (not related to spark reg), so this depends on whether the label is binary or continuous (i'm not sure about multi-class case), and it shouldn't depend on model type (unless it's multi-class classification).

Reply:
TODO: this part also needs to change if multi-class classification is considered for descaling

Reply:
Turns out for logistic regression, descaling only requires feature standard deviation, so this part stays the same.